src/type-translator.ts

```diff
@@ -145,6 +148,7 @@ export class TypeTranslator {
    *     as {?}.
    * @param symbolsToPrefix a mapping from symbols (`Foo`) to a prefix they should be emitted with
    *     (`tsickle_import.Foo`).
+   * @param isAmbient whether emitting happens in an ambient context.
```

Comment:
Uh, that param was removed in a later fixup but I forgot to drop the `@param`. "ambient context" == "any ancestor node has 'declare' on it"